### ISSUE TITLE
[feature request] Restore Pandoc's verbose output somehow?

### ISSUE THREAD

**Issue by iandol** [htdebeer/pandocomatic#108]
I understand you removed passing `pandoc` the `--verbose` option to not conflict with your internal communication. However Pandoc's verbose output can be useful / interesting during compilation. I wonder if there is some way of collecting the verbose output and restoring this output, without causing the bug you previously fixed?

**Comment by htdebeer** [comment]
The issue with `--verbhose` is not something that happens in a regular situation with the `--verbose` option. E.g, the following input runs fine:

```markdown
---
pandocomatic_:
  pandoc:
    from: markdown
    to: latex
    output: test.pdf
    verbose: true
...

Hello **World**!
```

It is easy to prevent by disabling the `--verbose` option. Because I haven't used this `--verbose` option myself, I assumed, apparently incorrectly, that this option would not be used often anyway.

However, from your message I gather that this option is used more often. If so, disabling isn't the best option, no.

Looking at paru, I see that I capture the error stream during the conversion, but only print it back to the error stream after the conversion is finished. That's clearly not an issue in simple situations like the example above. 

Reproducing the [original issue](#102), I see that it occurs when using filters or lua-filters. Somehow, the delayed printing of the verbose output, or the capturing of the output streams is interfering with how pandoc works here.

I'll look into this further, but it might take a while to investigate.

In the meanwhile, as a workaround, if you want to enable `--verbose` output, you can comment out lines 153-157 in the file [`lib/pandocomatic/command/convert_file_command.rb`](https://github.com/htdebeer/pandocomatic/blob/e16d8d60abce3adac41343faf9affac7e6d8ed7e/lib/pandocomatic/command/convert_file_command.rb#L153) and run pandocomatic via the test command like so:

```
/path/to/repository/pandocomatic/test/pandocomatic.rb -i yourfile.md -o output.ext
```

**Comment by iandol** [comment]
Thanks Huub for the workaround, take all the time you need and thanks for the consideration.

**Comment by htdebeer** [comment]
Actually, I didn't reproduce the issue. Pandoc failed because I didn't have the filter programs in the right location. Once I installed them properly, everything seems to work as expected.

In the meanwhile, pandoc, pandocomatic, and paru have been altered, which also makes reproducing the issue a bit hairy. Maybe @ppenguin can chime in to see if they can still reproduce the issue with the latest paru/pandocomatic/pandoc? Of course, they should then first enable the `--verbose` option like I explained in my previous message.

Alternatively, I could add a switch to pandocomatic to enable the `--verbose` option again. Something like `--enable pandoc-verbose`. This way, the `--verbose` option doesn't interfere with regular pandocomatic behavior, but you can switch it on if you so desire. Then, if switching on causes strange behavior, we can look into it further or ignore it.

As an added bonus, we can use this `--enable` feature to add and test new features.

**Comment by htdebeer** [comment]
Implemented feature toggle to enable pandoc's verbose mode. To use, use command-line option `--enable pandoc-verbose`. Feature will be available in pandocomatic version 2.0. You can already test it by cloning the repository and running `test/pandocomatic.rb` as the pandocomatic executable.
